### ISSUE TITLE
On Mac, “Open With” stopped working in 3.31.1 and later with error "Maximum call stack size exceeded"

### ISSUE THREAD

**Issue by tjluoma** [mifi/lossless-cut#656]
**Describe the bug**

Until recently I have been able to open an .mp4 file on the Mac in LosslessCut either by using `open -a Lossless Cut /path/to/file.mp4` or by right-clicking and choosing 'Open With' and then choosing "Lossless Cut" from the list of apps.

This stopped working very recently.

This seems to happen with any .mp4 I have, not just one particular file.

I can still edit the videos if I drag them to the main Lossless Cut window.

**To Reproduce**

1. Open file using right-click » Open With » Lossless Cut or `open -a LosslessCut /path/to/file.mp4"
2. Immediately get error message:

```
Unhandled Error
RangeError: Maximum call stack size exceeded
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)
    at openFile (/Applications/LosslessCut.app/Contents/Resources/app.asar/build/electron.js:28:28)


```

**Expected behavior**
Expected the file to be opened for editing

**Screenshots**

<img width="372" alt="CleanShot 2021-02-14 at 12 43 05@2x" src="https://user-images.githubusercontent.com/26965/107884297-35304a00-6ec2-11eb-907f-9495221e3879.png">

**Desktop (please complete the following information):**
- OS: macOS version 11.2.1 (20D74)
- MacBook Air (M1, 2020)

**Additional context**

Here is the information that Lossless Cut asked me to add:

```
No error

{
  "state": {
    "filePath": "",
    "externalStreamFiles": [],
    "mainStreams": [],
    "copyStreamIdsByFile": {},
    "cutSegments": [
      {}
    ],
    "rotation": 360,
    "shortestFlag": false
  },
  "platform": "darwin",
  "version": "3.32.0"
}
```

I will see if I can try old versions and find the most recent one where this worked.

**Comment by tjluoma** [comment]
FYI - It **worked in version 3.30.0** and **stopped working in 3.31.1**.

**Comment by mifi** [comment]
Thanks. Have now fixed this! will be out in the next ver

**Comment by tjluoma** [comment]
Confirmed fixed in version 3.33.1.

Thank you! 🌟 ❤️ 👍
